test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompaction_2_IT.java

```diff
@@ -130,6 +136,10 @@ public void testSplitCancelsExternalCompaction() throws Exception {
       // compaction above in the test. Even though the external compaction was cancelled
       // because we split the table, FaTE will continue to queue up a compaction
       client.tableOperations().cancelCompaction(table1);
+
+      // Verify that the tmp file are cleaned up
+      Wait.waitFor(() -> FindCompactionTmpFiles
+          .findTempFiles(getCluster().getServerContext(), tid.canonical()).size() == 1);
```

Comment:
Should this wait for zero?

```suggestion
          .findTempFiles(getCluster().getServerContext(), tid.canonical()).size() == 0);
```

Reply:
Good catch - copy/paste error. I found another instance of it too.

Reply:
Fixed in 9f4315a